activemodel/lib/active_model/translation.rb

```diff
@@ -22,7 +22,7 @@ module ActiveModel
   module Translation
     include ActiveModel::Naming
 
-    singleton_class.attr_accessor :raise_on_missing_translations
+    mattr_accessor :raise_on_missing_translations
```

Comment:
Should those be inheritable? Can a subclass of a model disable the option?

Reply:
I'd say a subclass should be able to change this option

Reply:
The receiver of `raise_on_missing_translations` method is a class. https://github.com/shouichi/rails/blob/a8699d5be2125f8ce921155e99f413362a75fbcc/activemodel/lib/active_model/translation.rb#L65 So, it works with inheritance by defining a class method. Please take a look at the added tests and changelog.

If there is a better/idiomatic way, please let me know.